pyfr/plugins/fluidforce.py

```diff
@@ -145,6 +177,15 @@ def __call__(self, intg):
 
             # Do the quadrature
             f[:ndims] += np.einsum('i...,ij,jik', qwts, p, norms)
+            if self.morigin:
+                rfpts = self._rfpts[etype, fidx]
+                # Pressure force moments
+                if self.ndims == 2:
+                    m[:self._mcomponents] += np.einsum(
+                        'i...,ij,ji', qwts, p, np.cross(rfpts, norms))
```

Comment:
Factor out the string and assign this based off an if statement.  Can also move the cross into the einsum expression to avoid a temporary.

Reply:
Done. I'll wait for a bit more validation before merging the cross product in the einsum (see the next comment)